packages/ui-components/src/components/modal/modal.tsx

```diff
@@ -0,0 +1,67 @@
+import { Component, Host, h, Prop, Event, EventEmitter } from '@stencil/core';
+import { EIconName } from '../icon/icon.types';
+import { IModalConfig, IModalEvents } from './modal.types';
+
+@Component({
+	tag: 'kv-modal',
+	styleUrls: {
+		night: 'modal.night.scss',
+		light: 'modal.light.scss'
+	},
+	shadow: true
+})
+export class KvModal implements IModalConfig, IModalEvents {
+	/** @inheritdoc */
+	@Prop() label?: string;
```

Comment:
Should this be named as `title` ?

Reply:
Doing that would set off alarms in stencil's linter since it's a reserved keyword in HTML, maybe `headerTitle` or something similar?

Reply:
`headerTitle` is good